### DISCUSSION TITLE
Renumber via API/webservice

### DISCUSSION THREAD

**Discussion by Dervis-vL** [Dlubal-Software/RFEM_Python_Client#356] [Q&A] [answered]
Is there a way to:
1. renumber using the API/webservice like in the GUI?
![image](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/65704977/212d75bb-5281-4786-b825-b65af237e4c1)

2. renumber the load cases and loads too?

Kind regards!

**Comment by heetrojivadiya** [comment, marked as answer]
> Is there a way to:
> 
> 1. renumber using the API/webservice like in the GUI?
>    ![image](https://private-user-images.githubusercontent.com/65704977/322809859-212d75bb-5281-4786-b825-b65af237e4c1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzNTY0NjAsIm5iZiI6MTcxMzM1NjE2MCwicGF0aCI6Ii82NTcwNDk3Ny8zMjI4MDk4NTktMjEyZDc1YmItNTI4MS00Nzg2LWI4MjUtYjY1YWYyMzdlNGMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE3VDEyMTYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM3ODc2YzNkOWJjMjg3MGIxODkyZDhkZjg4MjU4ZWY1NzZiZmNkNDZiMzA2ZmExODFmNWI2YjBlMjRkMzIzMTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fioTJKGnagu1Jpe_l-gqS91-Y4OKLbYCEfSpp5faKng)
> 2. renumber the load cases and loads too?
> 
> Kind regards!

Hello @Dervis-vL,

unfortunately, methods in `Tools` are currently not available in Webservice. So, any of the two are not possible with API/Webservice.

![image](https://github.com/Dlubal-Software/RFEM_Python_Client/assets/108512789/6bf270e5-5266-4b7c-99ae-cd3edb19b441)
